plugins/vendor/zoom/lib/kZoomFlowManager.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+* @package plugins.Vendor
+* @subpackage zoom
+*/
+
+class kZoomFlowManager implements kObjectCreatedEventConsumer
+{
+	const MAP_NAME = 'vendor';
+	const CONFIGURATION_PARAM_NAME = 'Zoom';
+	
+	
+	/**
+	 * @param BaseObject $object
+	 * @return bool
+	 */
+	public function objectCreated (BaseObject $object)
+	{
+		/* @var $captionAsset $object CaptionAsset */
+		/* @var $object CaptionAsset */
+		$zoomConfiguration = kConf::get(self::CONFIGURATION_PARAM_NAME, self::MAP_NAME);
+		$captionAsset = assetPeer::retrieveById($object->getId());
```

Comment:
This is useless , as you object is already caption asset.
You do the action on $object directly